node_modules/oae-util/lib/server.js

```diff
@@ -86,6 +86,16 @@ var setupServer = module.exports.setupServer = function(port, _config) {
         return next();
     });
 
+    // Trim whitespace on all post parameters
+    app.use(function(req, res, next) {
+        _.each(req.body, function(value, key) {
+            if (_.isString(value)){
+                req.body[key] = value.trim();
+            }
+        });
+        return next();
+    });
```

Comment:
Is there anything wrong with trying to create a link that has trailing spaces in its displayName, description, user list, .. and then verifying they're not there once the item is created?